lib/omniship/dhl.rb

```diff
@@ -13,5 +13,9 @@ def self.tracking_test?(tracking)
     def self.tracking_url(number)
       TRACKING_URL + number
     end
+
+    def self.bearer_token_required?
```

Comment:
could have created a base class for this method and inherited from it in each of the providers